forest/src/cli/mod.rs

```diff
@@ -451,19 +463,19 @@ mod test {
         ];
 
         for (input, expected) in cases {
-            assert_eq!(to_size_string(&input), expected.to_string());
+            assert_eq!(to_size_string(&input).unwrap(), expected.to_string());
         }
     }
 
     #[test]
     #[should_panic]
     fn to_size_string_negative_input_should_fail() {
-        to_size_string(&BigInt::from(-1i8));
+        to_size_string(&BigInt::from(-1i8)).unwrap();
```

Comment:
This might be more idiomatic:

```suggestion
        assert!(to_size_string(&BigInt::from(-1i8)).is_err());
```